### DISCUSSION TITLE
Version 6.1.0.0

### DISCUSSION THREAD

**Discussion by rsjaffe** [rsjaffe/MIDI2LR#781] [Announcements]
## New features
  * **Fine adjustments** This enables more precise adjustments of development parameters than were formerly available. There are new commands under the MIDI2LR develop submenu: 
     * Fine 2 
     * Fine 4
     * Fine 8
     * Fine 16
     * Fine Off
     * Fine Increase
     * Fine Decrease 
  
    that can be assigned to buttons. These buttons engage a "fineness" amount that affects all your adjustments until you cancel fineness. For example, pressing *Fine 4* will restrict the editing range to 1/4 of the usual range, centered, if possible, on the current value for all parameters on your controller. Pressing *Fine 4* when *Fine 4* is already in effect will cancel fineness. Pressing any other Fine number button when *Fine 4* is active will change the fineness value to the new number—for example *Fine 16* will reduce the editing range to 1/16 of the usual, again, centered on the current value for each parameter. This works the same for any of the Fine number commands: the first press engages that fineness, the second press cancels it, pressing any other fineness value switches to that one. You can also program a *Fine Off* button that will cancel fineness no matter what the current value is. Alternatively, you can use the *Fine Decrease* and *Fine Increase* buttons. Increase doubles fineness (2, 4, 8, 16, etc.) while *Fine Decrease* halves it. If you hit *Fine Decrease* when fineness is 2, it turns off the fineness adjustment.
  * **Quick develop** Added smaller increment for exposure: 1/6 stop.
## Improvements
  * App startup time reduced significantly.
  * App menus appear quicker.
## Bug fixes
  * **Crop adjustments** Interaction between different adjustments could, on occasion, cause loss of tracking of MIDI controller values, now fixed.

<hr /><em>This discussion was created from the release <a href='https://github.com/rsjaffe/MIDI2LR/releases/tag/'>Version 6.1.0.0</a>.</em>

**Comment by RiversGate** [comment]
I love the ability to adjust the "fineness" of an dial on my Midi Mal. I was wondering if there was a way to make it more uniform. For example, it would be nice if I could create a default fineness scale similar to what exists in LR. Example, with Exposure, each click of the dial would adjust  (+ or -)  0.10 increments. Highlights, shadows, whites, blacks (+ or -) 0.05. So the variable would be the increments. (+ or -) X .  X could be whatever the user wants it to be... 0.01 or 0.05 or 0.10 or whatever the user inserts as the X value.

**Reply by rsjaffe** [reply]
Currently, there's nothing like that and nothing planned. However, you can use the More--Less command (see https://github.com/rsjaffe/MIDI2LR/wiki/Commands#9) to emulate those keystrokes. If you have already been adjusting the parameter with a dial, then "More -- Less Last Modified" may work better.

**Comment by wowenkho** [comment]
Hello! Did you guys remove the 1/6 Exposure Adjustment in the MenuTrans.xml recently? 

I reinstalled the 6.1.0.0 just now and I saw that my configuration list is "Unassigned" on the buttons that I assigned exposure 1/6 increase and decrease to. 

Though the 1/6 exposure commands still work fine, the 1/6 adjustment isn't on the command list anymore. I also saw in the MenuTrans.xml in the source code zip that this is also the case.

![image](https://github.com/rsjaffe/MIDI2LR/assets/173887086/f7e23f0a-bba7-4f1e-903e-73c57a37d452)

![image](https://github.com/rsjaffe/MIDI2LR/assets/173887086/c5f2cbe5-518d-4e66-9f2e-42b8c2530015)

**Reply by wowenkho** [reply]
OK so yeah it kinda looks like the MenuTrans.xml in the recent downloadable packages doesn't show the 1/6 exposure adjustment commands. I luckily had another computer also with MIDI2LR and it had the 1/6 commands in its MenuTrans.xml. So I copy/pasted that.

**Comment by MMish75** [comment]
Hi. It may seem naive, but I'm trying to use the SMC Mixer as a Control Surface controller. I am at a loss as to how to set it up inside Studio One Pro v7
Anyone?
